scripts/run_tests.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
```

Comment:
This is basically the same as the python version -- I wrote it because the docker container didn't seem to be starting up properly in the python version, and the unit tests would fail and die in GHA before the narrative was even ready to go.